core/api/src/modules/ops/destination.ts

```diff
@@ -722,6 +699,28 @@ export namespace DestinationOps {
     }
   }
 
+  /**
+   * It's possible the run that created this export has been removed (it was from a deleting group).
+   */
+  async function safelyIncrementExportRunRuns(
```

Comment:
still might have a timing issue. you might want to catch "not found" or something if you really want it to be fully safe.